adot-testbed/app/src/test/resources/configurations/config-log4j.yaml

```diff
@@ -0,0 +1,19 @@
+receivers:
+  filelog:
+    include: [/logs/log4j.log]
+    start_at: beginning
+
+exporters:
+  logging:
+    verbosity: detailed
```

Comment:
Please remove this logging exporter. It will just create noise in the CI. Viewing the CW Logs for this should be easy enough where we don't have to rely on the logging exporter.

Reply:
Yes, more of this existence is to help in local testing to observe whether the logs exported or not. Removed from pipeline.